registry/storage/operators.go

```diff
@@ -225,5 +227,5 @@ func (s *operatorsStorage) DropOperators() error {
 
 // buildOperatorKey builds operator key using operatorsPrefix & index, e.g. "operators/1"
 func buildOperatorKey(id spectypes.OperatorID) []byte {
-	return bytes.Join([][]byte{operatorsPrefix, []byte(strconv.FormatUint(id, 10))}, []byte("/"))
+	return append(operatorsPrefix, []byte("/"+strconv.FormatUint(id, 10))...)
 }
```

Comment:
what was the issue with `bytes.Join`? joining bytes slices is kind of the purpose of it :)

also with `append` it becomes really ugly when you have to join more than 2 slices

Reply:
I agree. In general I think we shouldn't do things that are not related to the PRs scope. If you've encountered a change you think will benefit the code base, break it into a separate PR.

Reply:
@kchojn @y0sher i missed it before, but this breaks the key (slash was after now its before the id)

Reply:
@moshe-blox `bytes.Join` is much slower than append, `go` still has open performance issues with bytes.join;
The change was added at the beginning of this pr (when I also edited this file). Anyway, I make a revert of the change, you are right, its not part of this pr

Reply:
@kchojn i understand, but it's probably a difference on the order of nanoseconds and it's outside a hot path, so we're not too stressed to optimize it :)